src/server/utilities/mvtHeader/index.js

```diff
@@ -1,4 +1,4 @@
-import getMvtExperiments from './getMvtExperiments';
-import getMvtVaryHeaders from './getMvtVaryHeaders';
+import getServerExperiments from './getMvtExperiments';
+import getExperimentVaryHeaders from './getMvtVaryHeaders';
```

Comment:
Let's also rename these files to match
```suggestion
import getServerExperiments from './getServerExperiments';
import getExperimentVaryHeaders from './getExperimentVaryHeaders';
```